Kitodo-DataManagement/src/main/java/org/kitodo/data/elasticsearch/search/SearchRestClient.java

```diff
@@ -171,6 +167,26 @@ private String performRequest(HttpEntity entity, String httpMethod, String urlRe
         return output;
     }
 
+    private Map<String, String> getParameter() {
+        return getParameter(null, null);
+    }
+
+    private Map<String, String> getParameter(String offset, String size) {
```

Comment:
Please add some tests for this method. As behaviour is not covered if `offset` has some value and  `size` is null.